src/index.html

```diff
@@ -1,12 +1,430 @@
+            </div>
+            <p class="product-info__code">
+              Код товара: 195434
+            </p>
+          </div>
+          <div class="product-rating">
+            <div class="product-rating__stars stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <p class="product-rating__opinion">Отзывов: 5</p>
+          </div>
+          <div class="price">
+            <div class="price__section">
+              <p class="price__text">
+                Цена:
+              </p>
+              <p class="price__amount">
+                69 999 грн
+              </p>
+            </div>
+          </div>
+          <a
+            href="#"
+            class="button"
+            data-qa="card-hover"
+          >
+            купить
+          </a>
+        </section>
+        <section class="product-card catalog__card">
+          <img
+            src="images/imac.jpeg"
+            alt="image"
+            class="product-card__image"
+          >
+          <div class="product-info">
+            <div class="product-info__name">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </div>
+            <p class="product-info__code">
+              Код товара: 195434
+            </p>
+          </div>
+          <div class="product-rating">
+            <div class="product-rating__stars stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <p class="product-rating__opinion">Отзывов: 5</p>
+          </div>
+          <div class="price">
+            <div class="price__section">
+              <p class="price__text">
+                Цена:
+              </p>
+              <p class="price__amount">
+                69 999 грн
+              </p>
+            </div>
+          </div>
+          <a
+            href="#"
+            class="button"
+          >
+            купить
+          </a>
+        </section>
+        <section class="product-card catalog__card">
+          <img
+            src="images/imac.jpeg"
+            alt="image"
+            class="product-card__image"
+          >
+          <div class="product-info">
+            <div class="product-info__name">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </div>
+            <p class="product-info__code">
+              Код товара: 195434
+            </p>
+          </div>
+          <div class="product-rating">
+            <div class="product-rating__stars stars stars--4">
```

Comment:
What your stars block should do is enable you to change the number of stars in this line ` <div class="product-rating__stars stars stars--4">`  (for instance to 2 stars ` <div class="product-rating__stars stars stars--2">` ) and the card should show two active stars instead of 4)